packages/auth/package.json

```diff
@@ -37,9 +37,12 @@
     "@types/koa__cors": "^3.1.1",
     "@types/koa__router": "^8.0.11",
     "@types/uuid": "^8.3.4",
+    "@types/nock": "^11.1.0",
+    "@types/pino": "^7.0.5",
+    "@types/sinonjs__fake-timers": "^8.1.2",
```

Comment:
@njlie do we need these definitions with the newer jest?

Reply:
I think we can remove these.